examples/graph3d/13_tooltip_styling.html

```diff
@@ -0,0 +1,132 @@
+<!doctype html>
```

Comment:
This example is basically just a copy of [this](https://github.com/almende/vis/blob/develop/examples/graph3d/11_tooltips.html) one. Maybe it is better to adapt the other example for such a small change. What do you think?